.github/workflows/conformance-datapath.yaml

```diff
@@ -116,7 +116,11 @@ jobs:
       )) ||
       github.event_name == 'schedule' ||
       github.event_name == 'pull_request'
-    timeout-minutes: 45
+    strategy:
+      fail-fast: false
+      matrix:
+        kernel: ['5.4', '5.10', '5.15', 'bpf-next']
+    timeout-minutes: 120
```

Comment:
Doesn't look like `45 x 4`? Was `45` too much?

Reply:
I mean that GH can run this sequentially, right?

Reply:
These will be run in parallel. I set to 120min to catch some stability issues (TL;DR some VMs are still not using kvm-clock as a time source).